LibOS/shim/src/ipc/shim_ipc_pid.c

```diff
@@ -158,7 +158,7 @@ int ipc_pid_kill_callback (IPC_CALLBACK_ARGS)
             break;
     }
 
-    assert(ret != -ESRCH);
+    //assert(ret != -ESRCH);
```

Comment:
What is up with this assertion?  Is this a bad assert (should it be deleted?), or should it be re-enabled?